src/SFA.DAS.ApplyService.Web/Controllers/UsersController.cs

```diff
@@ -129,8 +129,6 @@ public async Task<IActionResult> PostSignIn()
                 return RedirectToAction("EnterApplicationUkprn", "RoatpApplicationPreamble");
             }
 
-            var organisation = await _organisationApiClient.GetByUser(user.Id);
```

Comment:
_organisationApiClient is not being used in this controller, please can you remove this code smell :)

Reply:
Done, and removed the other unnused dependencies too.